utils/grep.xml

```diff
@@ -3,36 +3,52 @@
-  <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
+  <description xml:lang="en">A Windows port of GNU Grep.
+
+  The latest port available from GnuWin32 is of version 2.5.4, which
+  was released in Feb 2009, quite some time ago.  Also, it always
+  annoyed me that color highlighting of Grep matches is not available
+  on MS-Windows.  This port of the latest release 2.10 of Grep fixes
+  that.
+
+  To use Perl-style regular expression, Grep needs to be built with
+  libpcre, so you have this as a bonus below.
+  </description>
+ <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.ico" type="image/vnd.microsoft.icon"/>
   <icon href="https://raw.githubusercontent.com/0install/0install.de-feeds/master/Gow.png" type="image/png"/>
   <category>Utility</category>
-  <homepage>http://gnuwin32.sourceforge.net/packages/grep.htm</homepage>
+  <homepage>https://sourceforge.net/projects/ezwinports/</homepage>
   <needs-terminal/>
-  <implementation arch="Windows-i486" id="sha1new=b09cab2cbcce8dc65848c0f0215adc1708bb60b4" langs="af be bg ca cs da de el es et eu fi fr ga gl he hr hu id it ja ko ky lt nb nl pl pt pt_BR ro ru rw sk sl sr sv tr uk vi zh_TW" license="GPL v3 (GNU General Public License)" released="2017-02-13" version="2.5.4-3">
-    <requires interface="http://repo.roscidus.com/devel/libintl">
-      <environment insert="bin" name="PATH"/>
-    </requires>
-    <requires interface="http://repo.roscidus.com/devel/gettext">
-      <environment insert="bin" name="PATH"/>
-    </requires>
-    <requires interface="http://repo.roscidus.com/lib/libiconv">
-      <environment insert="bin" name="PATH"/>
-    </requires>
-    <requires interface="http://repo.roscidus.com/lib/regex">
-      <environment insert="bin" name="PATH"/>
-    </requires>
-    <requires interface="http://repo.roscidus.com/lib/pcre">
-      <environment insert="bin" name="PATH"/>
-    </requires>
+  <group>
     <command name="run" path="bin/grep.exe"/>
     <command name="egrep" path="bin/egrep.exe"/>
     <command name="fgrep" path="bin/fgrep.exe"/>
-    <manifest-digest sha256new="2Y4YI2IHSPIICI7QRFKKXWEABHXHI3XP6RD75UKJVOLAFY5Z5ABA"/>
-    <archive href="https://sourceforge.net/projects/gnuwin32/files/grep/2.5.4/grep-2.5.4-bin.zip" size="451824" type="application/zip"/>
-    <archive href="https://github.com/kkeybbs/gnuwin32/blob/master/gnuwin32/grep-2.5.4-bin.zip?raw=true" size="451824" type="application/zip"/>
-  </implementation>
-  <package-implementation package="grep"/>
+    <implementation license="GPL v3 (GNU General Public License)" arch="Windows-i486" id="sha1new=b09cab2cbcce8dc65848c0f0215adc1708bb60b4" langs="af be bg ca cs da de el es et eu fi fr ga gl he hr hu id it ja ko ky lt nb nl pl pt pt_BR ro ru rw sk sl sr sv tr uk vi zh_TW" released="2009-02-13" version="2.5.4-3">
```

Comment:
I'd suggest removing the `langs` attribute as long as there are no other implementations with a different set of languages for 0install to potentially choose.

Reply:
The langs differ in that version 2.10 adds zh_CN